docs/en/solutions/How_to_Install_and_use_Featureform.md

```
@@ -0,0 +1,767 @@
+  However, when using Kubernetes Jobs to run `Featureform` tasks, image pull failures can cause task failures. Therefore, it's recommended to configure the correct image registry address.
+
+  The configuration field is as follows:
+
+  ```yaml
+  global:
+    repo: <Image Repository Address>/3rdparty/featureformcom
+
+  ```
+
+  How to obtain the `Image Repository Address`:
+  - In the `Administrator` view, check the `Image Repository Address` field under the `Overview` Tab page of the corresponding cluster details page.
```

markdownlint-cli2 (0.17.2)

Unordered list indentation
Expected: 0; Actual: 2

(MD007, ul-indent)